saadmob/src/main/java/tv/superawesome/plugins/publisher/admob/SAAdMobAdapter.kt

```diff
@@ -19,7 +19,7 @@ class SAAdMobAdapter : Adapter() {
         initializationCompleteCallback: InitializationCompleteCallback,
         mediationConfigurations: MutableList<MediationConfiguration>
     ) {
-        AwesomeAds.init(context, false)
+        AwesomeAdsSdk.init(context, Configuration())
```

Comment:
```suggestion
        AwesomeAdsSdk(context, Configuration())
```
Surely this is possible?

Reply:
Not really, it is not a class constructor.